src/App.tsx

```diff
@@ -1,23 +1,38 @@
-import React, {useEffect} from 'react';
+import React, { useEffect, useState } from 'react';
 import { getWeatherByLatLon } from "./weatherApi";
+import { googleMapsApi } from "./apis/googleMapsApi"
 import logo from './logo.svg';
 import './App.css';
 
 function App() {
-
+  const [searchString, setSearchString] = useState<string>("");
   useEffect(() => {
+
     getWeatherByLatLon(19.4150109, -98.1410678)
-        .then(res => console.log(res))
-        .catch(err => console.log(err));
+      .then(res => console.log(res))
+      .catch(err => console.log(err));
   }, [])
 
+
+  function onChangeEntry(dataString: string) {
+    setSearchString(dataString)
+  }
+
+  function getLatLon(searchString: string) {
```

Comment:
What do you think about use arrow functions?

`const getLatLon = (search: string) => googleMapsApi(search)`

Reply:
Sure, please check again, i already made the change.